src/components/Project.js

```diff
@@ -24,18 +23,20 @@ function Project(props) {
             props.index % 2 === 1 ? "right" : ""
           }`}
         >
-          <a src={props.project.deployment}>
-            <img
-              className={`project-img`}
-              src={props.project.imageURL}
-              alt={props.project.title}
-            />
-          </a>
+          <img
+            className={`project-img`}
+            src={props.project.imageURL}
+            alt={props.project.title}
+          />
         </div>
         <div className="project-text">
           <h2 className="project-title">{props.project.title}</h2>
-          <p className="project-description">{props.project.description}</p>
-          <div className="project-actions">
+          <div className="project-description">
+            {props.project.preview.split("\n").map((p, key) => {
+              return <p key={key}>{p}</p>;
+            })}
+          </div>
```

Comment:
Had to add for paragraph breaks since BE is serving one string for entire description